### DISCUSSION TITLE
Measures update

### DISCUSSION THREAD

**Discussion by alexis-bvq** [atoti/atoti#647] [General]
## Description

In order to win even more time for starting the cubes, we could add the folder and the formatting of the measure within the map of Measures.update().

**Comment by tibdex** [comment]
Hi Alexis, this would indeed be nice but currently measure folders and formatters can only be changed on instances of the `Measure` class which you can only get once the measure has already been published to the cube. What you suggest requires keeping track of the folders and formatters on the Python side before measures are published. This would add some complexity so the speed up gained from doing this would have to be worth it. Do you have some perf measurements to share?

**Comment by alexis-bvq** [comment]
I just did a use case where I change the folder of 300 measures, the cube took 2min30 to do that.

**Comment by alexis-bvq** [comment]
Hello, what is the status on this issue ? It could be critical for some projects that needs to start as fast as possible and it is a huge waste of starting time.
